sched/semaphore/sem_holder.c

```diff
@@ -713,9 +714,8 @@ void nxsem_boost_priority(FAR sem_t *sem)
  *
  ****************************************************************************/
 
-void nxsem_release_holder(FAR sem_t *sem)
+void nxsem_release_holder_tcb(FAR struct tcb_s *rtcb, FAR sem_t *sem)
```

Comment:
rtcb -> htcb 

(rtcb is commonly used as an abbreviation for running tcb, so the name is really confusing)

Reply:
That's right. It was a copy-paste, will change thanks.